src/components/TeamMember.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react';
+import AOS from 'aos';
+import 'aos/dist/aos.css';
+import '../TeamMember.css';
+
+export default function TeamMember(props) {
```

Comment:
lets extract props from here
```suggestion
export default function TeamMember({ isRight, name, image }) {
```